src/vnsw/opencontrail-vrouter-netns/opencontrail_vrouter_netns/vrouter_hyperv.py

```diff
@@ -14,7 +14,7 @@
 from common import validate_uuid
 
 
-def powershell(cmds):
+def call_powershell(cmds):
```

Comment:
I agree that such class would be pretty useful, however I don't think it's worth the effort just for one script.